ContentPatcher/docs/i18n/zh/author-guide/tokens.md

```diff
@@ -0,0 +1,2011 @@
+<td>
+
+是否完成献祭路线。可取值：`true` 或者 `false`。
+
+</td>
+<td><a href="#IsCommunityCenterComplete">#</a></td>
+</tr>
+
+<tr valign="top" id="IsJojaMartComplete">
+<td>IsJojaMartComplete</td>
+<td>
+
+是否完成JOJA路线。可取值：`true` 或者 `false`。
+
+</td>
+<td><a href="#IsJojaMartComplete">#</a></td>
+</tr>
+
+<tr valign="top" id="HavingChild">
+<td>HavingChild</td>
+<td>
+
+正在怀孕或领养孩子的玩家和NPC的姓名。玩家姓名用`@`做前缀避免与NPC重名。例如，检查当前玩家是否在生育孩子：
+
+```js
+"When": {
+    "HavingChild": "{{spouse}}"
+}
+```
+
+注意：
+* `"HavingChild": "@{{playerName}}"`和`"HavingChild": "{{spouse}}"`等效。
+* 另见 `Pregnant` 令牌。
+
+</td>
+<td><a href="#HavingChild">#</a></td>
+</tr>
+
+<tr valign="top" id="Pregnant">
+<td>Pregnant</td>
+<td>
+
+当前怀孕的玩家或NPC。这是 `HavingChild` 的子集，仅适用于异性关系中的女性伴侣（因为同性伴侣只能收养不能怀孕）。
+
+</td>
+<td><a href="#Pregnant">#</a></td>
+</tr>
+</table>
+<a name="number-manipulation"></a>
+
+### 数字操纵
+<table>
+<tr>
+<th>条件</th>
+<th>用途</th>
+<th>&nbsp;</th>
+</tr>
+
+<tr valign="top" id="Count">
+<td>Count</td>
+<td>
+
+获取令牌当前包含的值的数量。例如，`{{Count:{{HasActiveQuest}}}}`是任务列表里接取但未完成的任务数。
+
+</td>
+<td><a href="#Count">#</a></td>
+</tr>
+
+<tr valign="top" id="Query">
+<td>Query</td>
+<td>
+
+进行任意算术和逻辑运算，详见 [_查询表达式_](#query-expressions)。
+
+</td>
+<td><a href="#Query">#</a></td>
+</tr>
+
+<tr valign="top" id="Range">
+<td>Range</td>
+<td>
+
+最小值和最大值之间的所有整数（闭区间）。主要是用于比较数值，例如：
+
+```js
+"When": {
+   "Hearts:Abigail": "{{Range: 6, 14}}" //等同于"6, 7, 8, 9, 10, 11, 12, 13, 14"
+}
+```
+
+可在单个数值上使用令牌（比如`{{Range:6, {{MaxHearts}}}}`）或者一整个都用令牌（比如
+`{{Range:{{FriendshipRange}}}})`）只要符合`最小值, 最大值`的格式。
+
+为减少卡顿，不能超过5000个数字，应尽可能更少。
+
+</td>
+<td><a href="#Range">#</a></td>
+</tr>
+
+<tr valign="top" id="Round">
+<td>Round</td>
+<td>
+
+将输入数值近似为更少的小数位。
+
+默认情况下，四舍五入到最接近的整数。例如，
+`{{Round: 2.1 }}`结果是`2`，`{{Round: 2.5555 }}`结果是`3`。
+
+该令牌可以改变近似位数：
+
+用法 | 结果 | 含义
+----- | ------ | -----------
+`Round(2.5555)` | `3` | 近似为整数。
+`Round(2.5555, 2)` | `2.56` | 近似为两位小数。
+`Round(2.5555, 2, down)` | `2.55` | `up`向上取整，或者`down`向下取整。（不指定就默认[四舍五入](https://en.wikipedia.org/wiki/Rounding#Round_half_to_even)）。
+
+主要为了匹配[查询表达式](#query-expressions)。例如，怪物生命值必须为整数，因此将计算结果四舍五入到最近的整数：
+
+```js
+{
+  "Action": "EditData",
+  "Target": "Data/Monsters",
+  "Fields": {
+    "Green Slime": {
+      "0": "{{Round: {{Query: {{multiplier}} * 2.5 }} }}",
+    }
+  }
+}
+```
+
+可以在单个数值上用令牌（比如`{{Round: {{value}}, 2}}`）或同时用一整个令牌（比如`{{Round: {{Settings}}}}`，其中`{{Settings}}` = `2.5, 3, up`），只要最终输入符合上述形式。
+
+</td>
+<td><a href="#Round">#</a></td>
+</tr>
+</table>
+
+<a name="string-manipulation"></a>
+
+### 字符串操纵
+<table>
+<tr>
+<th>条件</th>
+<th>用途</th>
+<th>&nbsp;</th>
+</tr>
+
+<tr valign="top" id="Lowercase">
+<td id="Uppercase">Lowercase<br />Uppercase</td>
+<td>
+
+转换输入文本的大小写：
+
+<dl>
+<dt>Lowercase</dt>
+<dd>
+
+转为全小写。<br />例如：`{{Lowercase:It's a warm {{Season}} day!}}` &rarr; `it's a warm summer day!`
+
+</dd>
+<dt>Uppercase</dt>
+<dd>
+
+转为全大写：<br />例如： `{{Uppercase:It's a warm {{Season}} day!}}` &rarr; `IT'S A WARM SUMMER DAY!`
```

Comment:
Would it be worth a translator's note that these tokens have no effect on Chinese text, or would that be obvious to Chinese modders?